pkg/codec/utils.go

```diff
@@ -0,0 +1,225 @@
+package codec
+
+// [math/big.Int]: https://pkg.go.dev/math/big#Int
+// [time.Time]: https://pkg.go.dev/time#Time
+// [time.Unix]: https://pkg.go.dev/time#Unix
+// [time.Time.Unix]: https://pkg.go.dev/time#Time.Unix
```

Comment:
These should not be necessary. Do the bracketed types not work as-is? `[math/big.Int]` is the only one that looks unusual. It may need to be `[big.Int]`, but the others should work.